src/components/AnchorList/style.js

```diff
@@ -95,7 +95,9 @@ export const AnchorContentDes = styled.div`
   min-height: 643px;
 `
 export const AnchorItemsDes = styled.div`
-  height: calc(100% - 50px);
+  height: 588px;
```

Comment:
do not set height of content to something fix, let the content determine the height